data/src/main/java/com/yapp/data/repository/MemberRepositoryImpl.kt

```diff
@@ -0,0 +1,46 @@
+package com.yapp.data.repository
+
+import com.google.firebase.firestore.FirebaseFirestore
+import com.yapp.data.model.MemberModel
+import com.yapp.data.model.MemberModel.Companion.mapTo
+import com.yapp.data.util.memberRef
+import com.yapp.domain.model.MemberEntity
+import com.yapp.domain.repository.MemberRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.tasks.await
+import javax.inject.Inject
+
+
+class MemberRepositoryImpl @Inject constructor(
+    private val fireStore: FirebaseFirestore
+) : MemberRepository {
+
+    override fun setMember(memberEntity: MemberEntity): Flow<Boolean> {
+        return flow {
+            fireStore.memberRef()
+                .document(memberEntity.id.toString())
+                .set(memberEntity)
+                .await()
+
+            emit(true)
+        }
+    }
+
+    override fun getMember(id: Long): Flow<MemberEntity?> {
+        return flow {
+            val document = fireStore.memberRef()
+                .document(id.toString())
+                .get()
+                .await()
+
+            if (document.exists()) {
+                emit(document.toObject(MemberModel::class.java)?.mapTo())
```

Comment:
```suggestion
                emit(document.toObject(MemberModel::class.java)?.mapTo())
                return@flow
```
여기에 return 이 있어야 할듯 !? 안그러면 아래 emit 까지 같이 호출되니까요